WalkingChallenge/LoginViewController.swift

```diff
@@ -65,13 +65,11 @@ extension LoginViewController: FBSDKLoginButtonDelegate {
       alert(message: "Error logging in: \(error)", style: .cancel)
       return
     }
-    
-    if(FBSDKAccessToken.current() == nil){
+    if(result.isCancelled) {
+      alert(message: "Login Cancelled", style: .default)
```

Comment:
I dont think that a guard for this is proper as it is not a precondition, and the alert seems superfluous as we will redirect them to the login page.